shell/packages/sandstorm-ui-powerbox/powerbox-server.js

```diff
@@ -302,8 +369,8 @@ Meteor.publish("powerboxOptions", function (requestId, descriptorList) {
   // packed format and base64-encoded. The publish populates a pseudo-collection called
   // `powerboxOptions`. Each item has the following fields:
   //
-  //   _id: Object with two fields: `requestId` (as passed when subscribing) and `optionId` (a
-  //       unique identifier for the option).
+  //   _id: Unique identifier string.
```

Comment:
When I try to implement the ID as an object containing `optionId` and `requestId`, I get the error:

```
Error: Meteor does not currently support objects other than ObjectID as ids

```